scripts/modules/b2b-account/account-hierarchy.js

```diff
@@ -165,8 +165,6 @@ define(["modules/jquery-mozu", 'modules/api', "underscore", "hyprlive", "modules
             if (!isDescendantOfAssociatedAccount) {
                 //Check whether this is a current logged in users associated account.
                 isDescendantOfAssociatedAccount = require.mozuData('user').accountId === item.id;
-                //set "View Account" menu on current account.
-                item.canViewAccount = isDescendantOfAssociatedAccount && (self.isUserAdmin() || self.isUserPurchaser());
```

Comment:
I know I suggested this fix but it might not be null/undefined safe. I'll change it to be `= false` always.